### ISSUE TITLE
Need some way to monitor progress

### ISSUE THREAD

**Issue by kheffah** [DigitalSlideArchive/HistomicsTK#421]
I'm not sure where this should go, but there should be a way to mark a slide as "Done" and another way to mark it as "Reviewed". For groups working on a project, each person might be assigned a set of slides and these need to be reviewed by an expert. This requires some "check" (maybe a simple checkbox beside slide name?) to know if this is done and/or QC'd.

**Comment by jbeezley** [comment]
I think we'll need to work out some details on this before I can get started.  The simplest approach would be to just add a metadata key like `reviewed=True` and display that somehow in the UI. I suspect this won't provide a broad enough feature set though.  To help inform the implementation, which of the following items would be of interest?

1. Keeping track of who reviewed the slide and when.
2. Limiting who can review a slide or assigning a specific person.  Is the reviewing access distinct from admin/write access?
3. Requiring multiple people to review a slide.
4. Showing in the girder UI (rather than just in histomicstk) the review information.
5. Filtering by review metadata (list unreviewed slides, list slides assigned to me, list slides I have reviewed, etc.).  Should this be a global list or is filtering only necessary below a certain folder or collection?

**Comment by kheffah** [comment]
@jbeezley Yes, that sounds about right. I would suggest at least two metadata keys: `Annotated=True` and `Reviewed=True`. Here is what I think on your queries:

1. Yes, that's definitely a needed.
2.  Yes, and yes. Pathologists who review slides are not necessarily going to be the group admins.
3. This is also needed, as the same slide could go through multiple levels of review (crude, then fine-grained by more specialized expert).
4. I'm not sure about this. May be @cooperlab and @dgutman can provide an answer.
5. Absolutely! This is really needed, especially on the reviewer's side. In a project we are working on, we have 25+ users, each with different folders and slide so it gets really challenging to keep track of what was done and which slides have already been reviewed. I am creating a very crude SQL workaround for now, just to keep things going for the current project, but this would be really helpful/needed for future projects. 

> Should this be a global list or is filtering only necessary below a certain folder or collection?

I think it only make sense that it is below a certain folder/collection, because one person could be involved in multiple projects.

Thanks!
